### DISCUSSION TITLE
Cambio en el glosario para la traducción de la frase event handler

### DISCUSSION THREAD

**Discussion by NGHdeveloper** [reactjs/es.reactjs.org#843] [Q&A]
Deberíamos cambiar el glosario para la traducción de _event handler_ de manejador de eventos a controlador de evento puesto que es mas usado como término en informática, [controlador de evento](https://www.google.com/search?q=%22controlador+de+evento%22&client=firefox-b-d&ei=z8K2ZKm8GuPz1sQPyPC5yAg&ved=0ahUKEwjp38aw25iAAxXjuZUCHUh4DokQ4dUDCA4&uact=5&oq=%22controlador+de+evento%22&gs_lp=Egxnd3Mtd2l6LXNlcnAiFyJjb250cm9sYWRvciBkZSBldmVudG8iMgYQABgHGB4yBhAAGAcYHjIGEAAYBxgeMggQABgFGAcYHjIIEAAYCBgHGB4yCBAAGAgYBxgeSJwbUMESWMEScAF4AJABAJgBWKABWKoBATG4AQPIAQD4AQHCAggQABiABBiwA8ICCRAAGAcYHhiwA8ICBxAAGB4YsAPCAgkQABgIGB4YsAPiAwQYASBBiAYBkAYI&sclient=gws-wiz-serp) tiene cerca de 196.000 resultados en Google search y [manejador de evento](https://www.google.com/search?q=%22manejador+de+evento%22&client=firefox-b-d&ei=Etm2ZMjDG4Dy1sQPrp-d-Ag&ved=0ahUKEwjIlIfO8JiAAxUAuZUCHa5PB48Q4dUDCA4&uact=5&oq=%22manejador+de+evento%22&gs_lp=Egxnd3Mtd2l6LXNlcnAiFSJtYW5lamFkb3IgZGUgZXZlbnRvIjIGEAAYBxgeMgYQABgHGB4yBhAAGAcYHjIGEAAYBxgeMgYQABgHGB4yBRAAGIAESJ8UUIsMWIsMcAF4AZABAJgBUKABUKoBATG4AQPIAQD4AQHCAgoQABhHGNYEGLAD4gMEGAAgQYgGAZAGCA&sclient=gws-wiz-serp) tiene cerca de 19.300 resultados (véase que traduzco a manejador de evento y no a manejador de eventos (en plural), puesto que _event_ en inglés esta escrito en singular, si se busca como [glosario informático "manejador de evento"](https://www.google.com/search?q=glosario+inform%C3%A1tico+%22manejador+de+evento%22&client=firefox-b-d&ei=y9u2ZMHIDujz1sQP_-66iAI&ved=0ahUKEwiB2qea85iAAxXouZUCHX-3DiEQ4dUDCA4&uact=5&oq=glosario+inform%C3%A1tico+%22manejador+de+evento%22&gs_lp=Egxnd3Mtd2l6LXNlcnAiK2dsb3NhcmlvIGluZm9ybcOhdGljbyAibWFuZWphZG9yIGRlIGV2ZW50byIyBRAhGKABMgUQIRigATIFECEYoAEyBRAhGKABMgUQIRigAUiGE1DfCljfCnADeAGQAQCYAZIBoAGSAaoBAzAuMbgBA8gBAPgBAvgBAcICChAAGEcY1gQYsAPiAwQYACBBiAYBkAYC&sclient=gws-wiz-serp) tiene cerca de 116.000 resultados contra 46.100 resultados de [glosario informático "controlador de evento"](https://www.google.com/search?q=glosario+inform%C3%A1tico+%22controlador+de+evento%22&client=firefox-b-d&ei=dty2ZIWIFMvc1sQP0suPqAU&ved=0ahUKEwiFm_Lr85iAAxVLrpUCHdLlA1UQ4dUDCA4&uact=5&oq=glosario+inform%C3%A1tico+%22controlador+de+evento%22&gs_lp=Egxnd3Mtd2l6LXNlcnAiLWdsb3NhcmlvIGluZm9ybcOhdGljbyAiY29udHJvbGFkb3IgZGUgZXZlbnRvIjIFECEYoAEyBRAhGKABSM9LUOgIWItEcAR4AZABAJgByAGgAfQCqgEFMi4wLjG4AQPIAQD4AQL4AQHCAgoQABhHGNYEGLAD4gMEGAAgQYgGAZAGAw&sclient=gws-wiz-serp), o sea que la segunda opción es menor, pero esto tiene una trampa oculta, si se pluraliza la palabra evento a eventos (que no es la traducción literal de _event_ en singular), los guarismos cambian de forma significativa de 22.700 por parte de [manejador de eventos](https://www.google.com/search?q=glosario+inform%C3%A1tico+%22manejador+de+eventos%22&client=firefox-b-d&ei=nuC2ZIHePK321sQPzYKs4AM&ved=0ahUKEwiBpcjn95iAAxUtu5UCHU0BCzwQ4dUDCA4&uact=5&oq=glosario+inform%C3%A1tico+%22manejador+de+eventos%22&gs_lp=Egxnd3Mtd2l6LXNlcnAiLGdsb3NhcmlvIGluZm9ybcOhdGljbyAibWFuZWphZG9yIGRlIGV2ZW50b3MiMgUQIRigATIFECEYoAEyBRAhGKABMgUQIRigAUiWIVDrE1jrE3ABeAGQAQCYAZYBoAGWAaoBAzAuMbgBA8gBAPgBAcICChAAGEcY1gQYsAPiAwQYACBBiAYBkAYD&sclient=gws-wiz-serp) contra 265.000 de [controlador de eventos](https://www.google.com/search?q=glosario+inform%C3%A1tico+%22controlador+de+eventos%22&client=firefox-b-d&ei=EeC2ZOWHLfrd1sQPwNiasAI&ved=0ahUKEwil1Jqk95iAAxX6rpUCHUCsBiYQ4dUDCA4&uact=5&oq=glosario+inform%C3%A1tico+%22controlador+de+eventos%22&gs_lp=Egxnd3Mtd2l6LXNlcnAiLmdsb3NhcmlvIGluZm9ybcOhdGljbyAiY29udHJvbGFkb3IgZGUgZXZlbnRvcyIyBRAhGKABSJ4dUJMOWJMOcAF4AJABAJgBeqABeqoBAzAuMbgBA8gBAPgBAeIDBBgBIEGIBgE&sclient=gws-wiz-serp), ¡casi 12 veces más!, ademas que esta última búsqueda demuestra que empresas informáticas como [Microsoft](https://learn.microsoft.com/es-es/previous-versions/office/developer/sharepoint-2010/ee556558(v=office.14)), [Adobe](https://experienceleague.adobe.com/docs/workfront/using/basics/workfront-terminology-glossary.html?lang=es), [Angular](https://docs.angular.lat/guide/glossary), etc. así lo definen.
Como corolario cabe acotar que manejador no es una palabra que figure en la RAE y los correctores ortográficos (al menos el que aplica Firefox) la apuntan como error ortográfico, en su lugar la palabra controlador no conlleva ningún error (por ello es adoptada como traducción correcta por los grandes proyectos).
Por los cambios que habría que hacer en todo el proyecto no se preocupen, yo con mucho placer me hago de cargo de los mismos.

**Comment by carburo** [comment]
Cualquiera de los dos términos me parece bien. Que una palabra no exista en el DLE no significa que no sea correcta, lo común es que la terminología especializada que no es de uso general no se recoja en los diccionarios. El único matiz es que «controlador» corre el riesgo de ser un término sobrecargado al usarse para traducir términos informáticos distintos como _«controller»_ y _«handler»_. Aun así si «controlador de eventos» está más asentado, no veo inconveniente en el cambio.

**Reply by NGHdeveloper** [reply]
Lo tomo como tarea y lo cambio tanto en el glosario, como en todo el sitio, ¡Gracias!.

**Reply by NGHdeveloper** [reply]
@carburo ¡eureka!, por fin encontré la traducción exacta de la palabra _handler_ adoptada por la RAE para ["Tecnicismos, neologismos y extranjerismos en español"](https://revistas.rae.es/bilrae/article/view/218), en el archivo descargable indica que _handler_ se traduce al español como "manipulador", seguí investigando y observe que en idioma portugués la traducen exactamente de esa manera, esto se condice con lo que expresas en "El único matiz es que «controlador» corre el riesgo de ser un término sobrecargado al usarse para traducir términos informáticos distintos como «controller» y «handler»", y tienes toda la razón, si te parece conveniente, cambio el glosario de _event handler_ = controlador de evento y _event handler function_ = función controladora de evento, hacia  _event handler_ = manipulador de evento y _event handler function_ = función manipuladora de evento, ademas de agregar la palabra _handler_ = manipulador y corregir las traducciones pertinentes.

**Comment by Byron2016** [comment]
veo en el [Glosario](https://github.com/reactjs/es.react.dev/wiki/Glosario) que se debe colocar  `event handler  |  controlador de evento (m) \| controladora de evento (f)` pero x que no hacer lo mismo que lo propuesto con `root` y colocar `raíz(root)` o sea quedaría así: 

`event handler  |  controlador de evento(event handler) (m) \| controladora de evento(event handler)  (f)`

Hay expresiones como root, event handler que ya nos son más familiares en inglés que en español, pero creo que si se debe poner la traducción en español considerando a alguna persona que no entienda el concepto de la palabra inglesa, pero lo reforzamos con la palabra inglesa entre paréntesis.
